Examples/MAX78000/Hello_World/.vscode/settings.json

```diff
@@ -33,12 +33,12 @@
     "OCD_path":"${config:MAXIM_PATH}/Tools/OpenOCD",
     "ARM_GCC_path":"${config:MAXIM_PATH}/Tools/GNUTools/${config:v_Arm_GCC}",
     "xPack_GCC_path":"${config:MAXIM_PATH}/Tools/xPack/riscv-none-elf-gcc/${config:v_xPack_GCC}",
-    "Make_path":"${config:MAXIM_PATH}/Tools/MSYS2/usr/bin",
+    "Make_path":"${config:MAXIM_PATH}/Tools/GNUTools/Make",
+    "MSYS_path":"${config:MAXIM_PATH}/Tools/MSYS2",
```

Comment:
Just double checking, did you intend to take /usr/bin off the MSYS_Path?

Reply:
Yes, I added it up in the path setting itself

![image](https://github.com/Analog-Devices-MSDK/msdk/assets/38844790/d6311c29-9374-4c28-821e-4cae0ea3dc49)

Doing it this way in case I need to reference anything else out of the root MSYS folder in the future.